internal/usecases/cli/cli.go

```diff
@@ -125,10 +125,14 @@ func (au *UseCases) validateTextOutputFilePath(cfg *config.Config) error {
 
 func (au *UseCases) validateFilePathAndExtension(cfg *config.Config, extension string) error {
 	if filepath.Ext(cfg.JSONOutputFilePath) != extension {
-		return fmt.Errorf("%snot valid file of type %s", messages.MsgErrorJSONOutputFilePathNotValid, extension)
+		return fmt.Errorf("%s %s", messages.MsgErrorJSONOutputFilePathNotValidExtension, extension)
 	}
-	if output, err := filepath.Abs(cfg.JSONOutputFilePath); err != nil || output == "" {
-		return errors.New(messages.MsgErrorJSONOutputFilePathNotValid + err.Error())
+	output, err := filepath.Abs(cfg.JSONOutputFilePath)
+	if err != nil {
+		return fmt.Errorf("%s %s", messages.MsgErrorJSONOutputFilePathNotValidUnknown, err.Error())
+	}
+	if output == "" {
+		return fmt.Errorf("%s", messages.MsgErrorJSONOutputFilePathNotValidUnknown)
```

Comment:
Use `errors.New(messages.MsgErrorJSONOutputFilePathNotValidUnknown)` instead.

Reply:
is equal

Reply:
i beleave this isn't necessary

Reply:
I don't think that is equal, `fmt.Errorf` is for string formatting and create an error, and `error.New` is just for creating an error.